zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/job/JobUpdateProcessor.java

```diff
@@ -70,7 +70,7 @@ public void processRecord(final TypedRecord<JobRecord> command) {
 
     if (isJobUpdated) {
       stateWriter.appendFollowUpEvent(key, JobIntent.UPDATED, job);
-      responseWriter.writeEventOnCommand(key, JobIntent.UPDATED, job, command);
     }
+    responseWriter.writeEventOnCommand(key, JobIntent.UPDATED, job, command);
```

Comment:
❌ If we respond it to the user, we should also reflect it on the logstream with an event. Let's simply remove the `if (isJobUpdated)` statement.

Reply:
Thanks, @korthout, I absolutely agree with this 👍